backend/app/data/test/wikipedia_arkg.ttl

```diff
@@ -1,16 +1,25 @@
-@base <http://imlapps.github.io/anti-recommender/anti-recommendation/> .
+@prefix urn: <http://imlapps.github.io/anti-recommender/anti-recommendation/> .
+
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix schema: <http://schema.org/> .
 @prefix wd: <http://www.wikidata.org/entity/> .
 
-<Nikola_Tesla> rdf:type schema:Recommendation .
+urn:uuid:edf5a5c1-014e-4dfa-a503-8cbf1672aac8 rdf:type schema:Recommendation .
+urn:uuid:edf5a5c1-014e-4dfa-a503-8cbf1672aac8 schema:about wd:Q9036 .
+urn:uuid:edf5a5c1-014e-4dfa-a503-8cbf1672aac8 schema:itemReviewed wd:Q44666 .
+urn:uuid:edf5a5c1-014e-4dfa-a503-8cbf1672aac8 schema:itemReviewed wd:Q762 .
+
+wd:Q9036 rdf:type schema:Thing .
+wd:Q9036 schema:name "Nikola_Tesla" .
+wd:Q9036 schema:subjectOf schema:TextObject .
```

Comment:
schema:TextObject is a class, not an instance.

I'd suggest you model this off the schema.org on the Wikidata entity:
http://www.wikidata.org/entity/Q9036.ttl

```
<https://en.wikipedia.org/wiki/Nikola_Tesla> a schema:Article ;
	schema:about wd:Q9036 ;
	schema:inLanguage "en" ;
	schema:isPartOf <https://en.wikipedia.org/> ;
	schema:name "Nikola Tesla"@en ;
	wikibase:badge wd:Q17437798 .
```

You don't need the badge.

The main difference is that you're using pure schema.org and declaring wd:Q9036 a schema:Thing instead of a wikibase:Item.

Reply:
I have changed the ARKG to match the Wikidata model.